editor/find_in_files.cpp

```diff
@@ -54,9 +54,15 @@ inline void pop_back(T &container) {
 	container.resize(container.size() - 1);
 }
 
-static bool find_next(const String &line, String pattern, int from, bool match_case, bool whole_words, int &out_begin, int &out_end) {
+static bool find_next(const String &line, String pattern, int from, bool match_case, bool ignore_comments, bool whole_words, int &out_begin, int &out_end) {
 	int end = from;
 
+	if (ignore_comments) {
+		if (line.begins_with("#")) {
```

Comment:
This is still a problem, after you updated the commit, it only matches if the line has no tabbing.
Searching "clicked" correctly ignores the comment here:
```
func clicked():
	print("Clicked!")
# clicked
```

But not here:
```
func clicked():
	print("Clicked!")
# clicked
```

`line` is not being stripped out of tabs before the check here.